app/configurator/components/chart-options-selector.tsx

```diff
@@ -89,7 +90,14 @@ export const ChartOptionsSelector = ({
   });
 
   if (data?.dataCubeByIri) {
-    const meta = data.dataCubeByIri;
+    const meta = {
+      ...data.dataCubeByIri,
+      dimensions: [
+        ...data.dataCubeByIri.dimensions.filter(
+          (d) => !isNumericalDimension(d)
+        ),
+      ],
+    };
```

Comment:
Hmmm, I think that a right approach would be to have a `NumericalDimension` (identical to `Measure`?) and then to properly control the display in encodings, so we would have:

```
{
  field: "numericalFilter",
  optional: true,
  values: [
    "NumericalDimension",
  ],
  filters: true,
},
```

for new fields that make use of `NumericalDimensions` or simply skip it if it's not in the encoding. We also wouldn't need a `isNumerical` that is inside every dimension, even `GeoShapesDimension`. Btw – this would be really cool, even in case of the Population variable from the Covid dataset that triggered this change, to be able to have a range filter for such variables and e.g. show the data only for cantons of particular size.

I feel that this would be the right approach and I am not sure if we should move this logic to children at the moment, as anyway we would need to do some further refactoring? What do you think?

Reply:
You're right, a code comment is enough in this case, you can reference this github comment, I think. And create an issue so that we do not forget ?

Reply:
Done!